code/_helpers/text.dm

```diff
@@ -132,26 +132,32 @@
 
 //Used to strip text of everything but letters and numbers, make letters lowercase, and turn spaces into .'s.
 //Make sure the text hasn't been encoded if using this.
-/proc/sanitize_for_email(text)
+/proc/sanitize_for_account(text)
 	if(!text) return ""
 	var/list/dat = list()
-	var/last_was_space = 1
+	var/last_was_fullstop = 1
```

Comment:
What's going on here? Was the previous code lying?

Reply:
The behavior has changed slightly. Before spaces were turned to full stops, but you could not manually add full stops.  Now you can, so the name of the var has been changed to reflect that the actual character in the output is a full stop,